shell/initialize/App.js

```diff
@@ -7,19 +7,15 @@ import NuxtError from '../layouts/error.vue';
 import NuxtLoading from '../components/nav/GlobalLoading.vue';
 
 import '../assets/styles/app.scss';
-import { getLayouts } from './layouts';
-
-const layouts = getLayouts();
 
 export default {
   render(h) {
     const loadingEl = h('NuxtLoading', { ref: 'loading' });
 
-    const layoutEl = h(this.layout || 'nuxt');
     const templateEl = h('div', {
       domProps: { id: '__layout' },
       key:      this.layoutName
-    }, [layoutEl]);
+    }, [h('nuxt')]);
```

Comment:
We still require the nuxt element which is a thin wrapper around <router-view>. I ran into some issues which I no longer recall when switching to router-view. I'm guessing <nuxt> does some component registration for fetch etc.